packages/database/src/core/Repo.ts

```diff
@@ -458,13 +458,19 @@ export function repoGetValue(repo: Repo, query: QueryContext): Promise<Node> {
   }
   return repo.server_.get(query).then(
     payload => {
-      const node = nodeFromJSON(payload as string);
+      const node = nodeFromJSON(payload as string).withIndex(
```

Comment:
Can we add a test for this? I assume this would be mostly a copy&paste from the code snippet.

Reply:
Discussed offline. This test requires adding an `.indexOn` to the rules and fitting this into the existing query test suite is sort of awkward rn.